content/building/access-private-git-submodules.md

```diff
@@ -4,19 +4,34 @@ title: Accessing private repositories
 weight: 4
 ---
 
-If your project requires accessing any private Git submodules or dependencies, you'll need to grant Codemagic access to them in order to build successfully. 
-You can do that by saving the SSH key to your repository as a secure [environment variable](https://docs.codemagic.io/building/environment-variables/), i.e. `SSH_KEY`. All environment variables whose name has the suffix `_SSH_KEY` will be automatically added to the SSH agent and will be ready for use during the whole build process.
+If your project requires accessing any private Git submodules or dependencies, you'll need to grant Codemagic access to them in order to build successfully.
 
-1.  Save the SSH key (e.g. `SSH_KEY`) for accessing the repository as an environment variable. Make sure to check **Secure**.
+1. [Create a SSH key pair](#generating-a-ssh-key) for use with Codemagic. Note that the SSH key **cannot** be password-protected.
+2. Add the **public key** to your repository settings.
```

Comment:
https://help.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account

https://confluence.atlassian.com/bitbucket/use-access-keys-294486051.html

https://docs.gitlab.com/ee/ssh/README.html#adding-an-ssh-key-to-your-gitlab-account